### PR TITLE
Handle missing add-on blockID.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: amo2kinto/exporter.py [hunk 1]
```diff
@@ -83,16 +83,17 @@ def write_addons_items(xml_tree, records, app_id, api_ver=3):
...
+                    # compare the numeric value i45 < i356.
+                    current_blockID = item['blockID'][1:]
+                    previous_blockID = emItem.attrib['blockID'][1:]
+                    # Group by and keep the biggest blockID in the XML file.
+                    if current_blockID > previous_blockID:
+                        emItem.attrib['blockID'] = item['blockID']
             else:
                 emItem = etree.SubElement(emItems, 'emItem',
-                                          blockID=item['blockID'])
+                                          blockID=item.get('blockID', item['id']))
```
Comment (leplatrem) [inline amo2kinto/exporter.py]: we have `emItem.set('id'` a few lines below, it's hard to follow without comment

File: amo2kinto/exporter.py [hunk 2]
```diff
@@ -83,16 +83,17 @@ def write_addons_items(xml_tree, records, app_id, api_ver=3):
...
             if item['guid'] in groupby:
                 emItem = groupby[item['guid']]
-                # Remove the first caracter which is the letter i to
-                # compare the numeric value i45 < i356.
-                current_blockID = item['blockID'][1:]
-                previous_blockID = emItem.attrib['blockID'][1:]
-                # Group by and keep the biggest blockID in the XML file.
-                if current_blockID > previous_blockID:
-                    emItem.attrib['blockID'] = item['blockID']
+                if 'blockID' in item:
```
Comment (leplatrem) [inline amo2kinto/exporter.py]: A small comment explaining that in the new system `blockID` is not mandatory, but `id` is.

File: amo2kinto/exporter.py (hunk 1 above)
Comment (Natim) [inline amo2kinto/exporter.py]: I don't understand this comment, the item id is the guid (add-on id not the kinto blocklist id)
